bin/commands/init/generate/.errors

```diff
@@ -0,0 +1,3 @@
+ZWEL0143E|143|Cannot find data set member %s. You may need to re-run zwe install.
+ZWEL0157E|157|%s (%s) is not defined in Zowe YAML configuration file.
+ZWEL0318E|318|Configuration stored in PARMLIB must use member name ZWEYAML when using generate action.
```

Comment:
This error code also shows up in configmgr: https://github.com/zowe/zowe-common-c/blob/be02654f1bb7753c0f183927837e4c4ee12b42f1/c/configmgr.c#L697 

This was confusing when I ran an init command with a bad defaults.yaml file (missing path) and got an error response.

Reply:
Related, if there's any error in the defaults.yaml - invalid syntax or path - the `init` command group tries to execute and submit ZWEGENER anyway. It looks like if there's an error reading YAML in configmgr.c, it returns NULL and the calling code can't differentiate between a valid, empty document and an error condition, so everything returns status 0 all the way up.

```
./bin/zwe init --configmgr -c zowe.test.yaml --dry-run
ZWEL0318E - Couldn't parse file '/ZOWE/zwe-integration/files/defaults.yaml': while parsing a block mapping at line 30, column 3, did not find expected key at line 32, column 5.
Temporary directory '/tmp/.zweenv-0000' created.
Zowe will remove it on success, but if zwe exits with a non-zero code manual cleanup would be needed. 
===============================================================================
>> CONFIGURE ZOWE 

....and so on
```